test/viewmodels/SummaryCardHelperSpec.scala

```diff
@@ -451,6 +451,35 @@ class SummaryCardHelperSpec extends SpecBase with ImplicitDateFormatter {
           classes = "govuk-tag "
         )
       }
+      "a financial penalty has been added and the user has paid - appealStatus Accepted" in {
+        val result = helper.tagStatus(None,Some(sampleLatePaymentPenaltyAppealedAccepted))
+        result shouldBe Tag(
+          content = Text(cancelledTag),
+          classes = "govuk-tag "
+        )
+      }
+      "a financial penalty has been added and the user has paid - appealStatus Reinstated " in {
+        val result = helper.tagStatus(None,Some(sampleLatePaymentPenaltyAppealedReinstated))
+        result shouldBe Tag(
+          content = Text(overdueTag),
+          classes = "govuk-tag penalty-due-tag"
+        )
+      }
+      "a financial penalty has been added and the user has paid - appealStatus Rejected" in {
+        val result = helper.tagStatus(None,Some(sampleLatePaymentPenaltyAppealedRejectedLPPPaid))
+        result shouldBe Tag(
+          content = Text(paidTag),
+          classes = "govuk-tag "
+        )
+      }
+
+      "a financial penalty has been added and the user is due - appealStatus Rejected" in {
```

Comment:
Could you just change the wording for this test - 
```suggestion
      "a financial penalty has been added and the user has not paid the penalty - appealStatus Rejected" in {
```